example/ios/Runner/AppDelegate.swift

```diff
@@ -1,33 +1,13 @@
-import background_locator
-import Flutter
-import path_provider
 import UIKit
-
-func registerPlugins(registry: FlutterPluginRegistry) {
-    GeneratedPluginRegistrant.register(with: registry)
-}
+import Flutter
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
-    override func application(
-        _ application: UIApplication,
-        didFinishLaunchingWithOptions launchOptions: [UIApplication
-            .LaunchOptionsKey: Any]?
-    ) -> Bool {
-        GeneratedPluginRegistrant.register(with: self)
-        BackgroundLocatorPlugin.setPluginRegistrantCallback(registerPlugins)
-
-        registerOtherPlugins()
-
-        return super
-            .application(application,
-                         didFinishLaunchingWithOptions: launchOptions)
-    }
-
-    func registerOtherPlugins() {
```

Comment:
Why did you remove this?

Reply:
It was removed due to running `flutter create .` in example when iOS Build was failing. I will add it again

Reply:
Checked out the documentation and example for path_provider and to me it seems manually registering the plugin is not necessary however I don't know anything about iOS dev so I can add it if it's needed

Reply:
@mehdok

Reply:
It is necessary for using other plugins in feedback. Same as Android.

Reply:
@mehdok Updated in latest commit. Please review further